src/conv/problem_description.cpp

```diff
@@ -176,6 +198,17 @@ void ProblemDescription::Serialize(std::ostream& stream) const
         // Group count > 1 identifies Group/Depthwise modes.
         if(GetGroupCount() != 1)
             optional << 'g' << GetGroupCount();
```

Comment:
Readability of db key: Remove `'_' <<` from line 216 and:
```suggestion
            optional << '_g' << GetGroupCount();
```

Reply:
updated

Reply:
[resolved]